docs/cases/api/actions-api/cases-actions-api-intro.asciidoc

```diff
@@ -15,7 +15,9 @@ which stores the information required to interface with the external system.
 
 {es-sec} uses these external APIs to send cases:
 
-* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]
+* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/rome/rest/c_ImportSetAPI[Import Set API]
++
+IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier use the https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]. They are marked as deprecated after you upgrade to version 7.16.0 or later. To ensure you have access to connector enhancements, such as updating incidents, we strongly recommended that you <<post-upgrade-deprecated-sn-connector, update deprecated connectors>> or create new ones. 
```

Comment:
I want to emphasize that from within Cases you can update incidents with the old connector and prior to 7.16. In 7.16, with the new connectors, we introduced the ability to update incidents from within Rules (prior to 7.16 it was available). I am not sure if this is clear to our users.